.github/workflows/generator_generic_slsa3.yml

```
@@ -107,7 +107,7 @@ jobs:
       - name: Create and sign provenance
         id: sign-prov
         shell: bash
-        # NOTE: Inputs and github context are set to environment variables in
+        # NOTE: Inputs and GitHub context are set to environment variables in
```

```suggestion
        # NOTE: Inputs and `github` context are set to environment variables in
```
The casing is intentional here as the `github` context is usually referred to in lowercase in GitHub docs.
https://docs.github.com/en/actions/learn-github-actions/contexts#github-context

I suggest adding ` to it to make it clearer what's being referred to.